src/sidebar/views/reports.ts

```diff
@@ -248,18 +248,18 @@ export class ReportsView implements TreeDataProvider<IssueMetadata> {
         // Second level, repr steps
         const currentStep = steps[element.reprStep];
 
-        const currentStepPath = currentReport.file.original_path;
-        const currentStepFile = basename(currentStepPath);
+        const currentStepFilePath = currentStep.file.original_path;
+        const currentStepFileName = basename(currentStepFilePath);
 
         const item = new TreeItem(
-            `${element.reprStep + 1}. [${currentStepFile}:${currentStep.line}] - ${currentStep.message}`
+            `${element.reprStep + 1}. [${currentStepFileName}:${currentStep.line}] - ${currentStep.message}`
         );
-        item.tooltip = `Full path to file: ${currentStepPath}`;
+        item.tooltip = `Full path to file: ${currentStepFilePath}`;
         item.collapsibleState = TreeItemCollapsibleState.None;
         item.command = {
             title: 'jumpToStep',
             command: 'codechecker.editor.jumpToStep',
-            arguments: [this.currentFile, element.entryIndex, element.reprStep, true]
+            arguments: [currentStepFilePath, element.entryIndex, element.reprStep, true]
```

Comment:
The jumpToStep command uses the bug's path as the base filename, not the repr. step's - so clicking doesn't jump to the correct location when step's (going to be) in a different file.

(This should probably be in a more descriptive param name or similar, surprised it's not...)

Reply:
If I understand correctly `jumpToStep` first parameter should be the file path from the report itself and not the file path from the bug step. I changed it back to `this.currentFile`.

Reply:
Yep, that's correct.